Src/Campus.Master.API/appsettings.json

```diff
@@ -9,5 +9,8 @@
   "AllowedHosts": "*",
   "Endpoints": {
     "QueryItemsLimit": 20
+  },
+  "ConnectionStrings": {
+    "Default": "Data Source=(LocalDB)\\MSSQLLocalDB;Database=CampusDb;Integrated Security=True;MultipleActiveResultSets=True"
```

Comment:
Change also connection strings for the dev appsettings file. And remove explicit connection strings declaration from prod mode file, because it's sensitive information, we'll keep it in secrets manager

Reply:
Moved